src/Func.cpp

```diff
@@ -2417,14 +2398,12 @@ void Func::realize(Realization dst, const Target &target) {
             << "\" has type " << func.output_types()[i] << ".\n";
     }
 
-    // In case these have changed since the last realization
-    compiled_module.set_error_handler(error_handler);
-    compiled_module.set_custom_allocator(custom_malloc, custom_free);
-    compiled_module.set_custom_do_par_for(custom_do_par_for);
-    compiled_module.set_custom_do_task(custom_do_task);
-    compiled_module.set_custom_trace(custom_trace);
-    compiled_module.set_custom_print(custom_print);
+    JITFuncCallContext jit_context(jit_handlers, jit_user_context);
+
+    // TODO: Figure this out.
+#if 0
```

Comment:
I'm working on a replacement which sets the cache size for the entire shared runtime. Should be up today.